### DISCUSSION TITLE
Importing models *solved*

### DISCUSSION THREAD

**Discussion by viveksood97** [cryinkfly/Autodesk-Fusion-360-for-Linux#204] [Q&A] [answered]
The file system is different when I try to import from computer. I don't know how to find my cad model while importing. Can someone tell me how to import files from computer to fusion 360?
![Screenshot from 2022-03-14 21-53-40](https://user-images.githubusercontent.com/48154838/158290363-816af37e-8eb8-4771-b0c1-553fc88e125e.png)

**Comment by cryinkfly** [comment, marked as answer]
@viveksood97 So you have to copy your models into your Wineprefix first, since Autodesk Fusion 360 is set up in a kind of sandbox!

That means you do the following:

1.) Select your file/s that you want to copy and then switch to your Wineprefix. For example: _/home/steve/.wineprefixes/fusion360/drive_c/users/steve/Documents/_

2.) Then switch to this folder in the file manager (Autodesk Fusion 360):

![filemanager-import](https://user-images.githubusercontent.com/79079633/158323696-6da02152-05e9-4018-8745-717eb59bc507.jpg)

3.) Import your file/s.
